src/main/kotlin/blackjack/view/InputView.kt

```diff
@@ -16,6 +16,16 @@ object InputView {
         }
     }
 
+    fun askPlayerBet(name: String): Int {
+        println()
+        println("Enter $name’s betting amount:")
```

Comment:
I think we could reduce the number of `println()` calls here while still achieving the same visual result. You've done it in other parts of view logic, so it'll be great to apply them everywhere.

It’s not critical right now, but the reason `why minimizing println() calls can be helpful` is pretty interesting. I recommend looking it up when you get a chance!